### ISSUE TITLE
Phalcon\Mvc\Model\Query\Builder BUG

### ISSUE THREAD

**Issue by cq-z** [phalcon/cphalcon#11487]
Phalcon\Mvc\Model\Query\Builder  
line 541 and line 600.

```
let currentBindTypes = this->_bindTypes;
            if typeof currentBindParams == "array" {
                let mergedTypes = currentBindTypes + bindTypes;

```

"currentBindParams" and  "currentBindTypes" ????

**Comment by sergeyklay** [comment]
Fixed in the `3.4.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.

**Comment by cq-z** [comment]
thank you
